static/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -434,6 +436,25 @@ class SpanDetail extends React.Component<Props, State> {
     };
   }
 
+  scrollBarIntoView = (spanId: string) => (e: React.MouseEvent<HTMLAnchorElement>) => {
+    // do not use the default anchor behaviour
+    // because it will be hidden behind the minimap
+    e.preventDefault();
+
+    const hash = `#span-${spanId}`;
+
+    this.props.scrollToHash(hash);
+
+    // TODO(txiao): This is causing a rerender of the whole page,
+    // which can be slow.
+    //
+    // make sure to update the location
+    browserHistory.push({
```

Comment:
Why are you updating the location if this is called with callbacks? Is it highlighted?

Reply:
This is for updating the hash URL.

Reply:
Yes, this is so that the hash is updated into the URL so it can be shared.